colcon-build-and-test/action.yaml

```diff
@@ -71,17 +71,13 @@ runs:
     - name: Test
       run: |
         . /opt/ros/${{ inputs.rosdistro }}/setup.sh
-        colcon lcov-result --initial --packages-above ${{ inputs.target-packages }}
+        colcon lcov-result --initial --verbose --packages-above ${{ inputs.target-packages }}
```

Comment:
What's this `--verbose` for?

Reply:
`--verbose` prints the coverage on the stdout. This makes debugging easier and allows us to see the results on the Actions page.

```
❯ colcon lcov-result --packages-select autoware_auto_common --verbose
Starting >>> autoware_auto_common
Reading tracefile /home/shima/repos/autoware.universe/build/autoware_auto_common/coverage.info
Summary coverage rate:
  lines......: 100.0% (99 of 99 lines)
  functions..: 100.0% (27 of 27 functions)
  branches...: 15.9% (72 of 452 branches)
Reading tracefile /home/shima/repos/autoware.universe/build/autoware_auto_common/coverage.info
Summary coverage rate:[autoware_auto_common - 0.6s]
  lines......: 100.0% (99 of 99 lines)
  functions..: 100.0% (27 of 27 functions)
  branches...: 15.9% (72 of 452 branches)
Finished <<< autoware_auto_common [0.61s]          

Summary: 1 package finished [1.07s]

Calculating total coverage... 
Reading tracefile /home/shima/repos/autoware.universe/lcov/total_coverage.info
Summary coverage rate:
  lines......: 100.0% (99 of 99 lines)
  functions..: 100.0% (27 of 27 functions)
  branches...: 15.9% (72 of 452 branches)

Generating HTML: Done
```

Reply:
It's nice!